### PR TITLE
Added 'hres_eccb.py' and 'hres_ic.py' to the repo

### LINKED ISSUE #5: Add `hres_eccb.py` and `hres_ic.py` python scripts to the repo
The `hres_eccb.py` and `hres_ic.py` are used to call and setup the current python scripts present in the repo.
They should be part of the repo as well. 

- [ ] Add `hres_eccb.py` and `hres_ic.py` python scripts to the repo

### PR DESCRIPTION
**Pull request by atteggiani**
Fixes #5 

Added scripts versions from `u-dh821`.

### CODE REVIEW

**Comment by penguian** [comment]
After adding these scripts to the repository, we must also ensure that this version of the scripts is used in the Regional Nesting Suite (e.g. `u-dh821`), whether through the current [install_cold](https://code.metoffice.gov.uk/trac/roses-u/browser/d/h/8/2/1/trunk/app/install_cold/opt/rose-app-nci-gadi.conf) method, or via a Conda environment. This would also involve deleting the scripts from [nci_hres_eccb](https://code.metoffice.gov.uk/trac/roses-u/browser/d/h/8/2/1/trunk/app/nci_hres_eccb/bin/hres_eccb.py) and [nci_hres_ic](https://code.metoffice.gov.uk/trac/roses-u/browser/d/h/8/2/1/trunk/app/nci_hres_ic/bin/hres_ic.py) respectively.

**Comment by atteggiani** [comment]
Yes, I think that is a crucial step, but could be dealt with at the end of the scripts testing/refactoring.
In this repo, I would concentrate to test all of the scripts and then once we are happy with everything, we can ensure they are effectively the ones used by the Regional Nesting Suite (RNS). 

I think the easiest way to achieve that is to add a task in the RNS that downloads the script directly from this Github repo, before executing it. 
What do you think?

File: src/hres_eccb.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.01/bin/python
```
Comment (penguian) [inline src/hres_eccb.py:1]: Remove the explicit path to Python and replace it with (e.g.) 
```
#!/bin/env python3
```
as appropriate.

File: src/hres_ic.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+#!/g/data/hh5/public/apps/miniconda3/envs/analysis3-23.01/bin/python
```
Comment (penguian) [inline src/hres_ic.py:1]: Remove the explicit path to Python and replace it with (e.g.)
```
#!/bin/env python3
```
as appropriate.

**Review by penguian — CHANGES_REQUESTED** [review]
Please address my comments on explicit Python paths.

File: src/hres_eccb.py (hunk 1 above)
Comment (atteggiani) [reply src/hres_eccb.py:1]: Good point, going to replace with `#!/usr/bin/env python3`

File: src/hres_ic.py (hunk 2 above)
Comment (atteggiani) [reply src/hres_ic.py:1]: As above

**Comment by atteggiani** [comment]
> Please address my comments on explicit Python paths.

Addressed your comments.

**Review by penguian — APPROVED** [review]
Approved. We can deal with PEP 8 and installation in later stages.

**Pull request merged**